src/hooks/app-data-provider/useWalletBalances.tsx

```diff
@@ -54,6 +54,6 @@ export const useWalletBalances = () => {
   return {
     walletBalances: aggregatedBalance,
     hasEmptyWallet,
-    loading: !walletBalances.length || !reserves.length,
```

Comment:
I think, since the app is in maintenance mode,  it's wiser to revert all the changes that we introduced just to support "empty" markets. We will test the whole integration on the forked chain.